integration_tests/src/main/python/string_test.py

```diff
@@ -779,4 +779,13 @@ def test_like_complex_escape():
                 'a like "\\%SystemDrive\\%\\\\\\\\Users%"',
                 'a like "_oo"'),
             conf={'spark.sql.parser.escapedStringLiterals': 'true'})
- 
+
+
+@pytest.mark.parametrize('from_base', [10], ids=idfn)
+@pytest.mark.parametrize('to_base', [16], ids=idfn)
+@pytest.mark.parametrize('pattern', ['[0-9]{1,18}', '[0-9a-fA-F]{1,15}'])
+def test_conv_dec_to_hex(from_base, to_base, pattern):
```

Comment:
What about ANSI mode tests?

Reply:
sounds good. I tested manually  the CPU with 3.3.2 some time ago and the behavior is identical there.

Reply:
When I look at the code for 3.4.0 I see that they do support ANSI mode and throw an exception on overflow.  We might need to fall back in some cases.